controls.cpp

```diff
@@ -209,6 +209,8 @@
 #include "netplay.h"
 #endif
 
+#include "libretro\libretro.h"
```

Comment:
wrong slash

Reply:
also shouldn't this be behind an `#ifdef __LIBRETRO__`

Reply:
Took out that line since don't need it anymore. But! I didn't realize we could add #ifdef __LIBRETRO__ also everywhere. Double d'oh!

Thanks for pointing all these things out!